src/pages/Register.tsx

```diff
@@ -0,0 +1,125 @@
+import { useState } from "react";
+import { Box, Button, Typography, TextField } from "@mui/material/";
+import { DatePicker, LocalizationProvider } from "@mui/x-date-pickers/";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+import axios from "axios";
+import { useAuth0 } from "@auth0/auth0-react";
+import { useUserContext } from "../context/UserContext";
+
+const styles = {
+  container: {
+    alignItems: "center",
+    display: "flex",
+    flexDirection: "column",
+    gap: 2,
+    height: "100%",
+    paddingTop: 5,
+  },
+  datePicker: {
+    width: 253.4,
+  },
+  title: {
+    textAlign: "center",
+    fontWeight: 700,
+    fontSize: 32,
+  },
+  submitButton: {
+    borderRadius: 5,
+    width: 253.4,
+  },
+};
+
+const Register = () => {
+  const { getAccessTokenSilently } = useAuth0();
+  const [username, setUsername] = useState("");
+  const [displayName, setDisplayName] = useState("");
+  const [birthDate, setBirthDate] = useState<Date | undefined>();
+  const { user, setUser } = useUserContext();
+
+  const submitUserInfo = async (e: React.SyntheticEvent) => {
+    e.preventDefault();
+    const token = await getAccessTokenSilently();
+    try {
+      setUser({ ...user, isLoading: true });
+      const newUserInfo = await axios.put(
+        `http://localhost:3001/api/users/${user.userId}`,
+        {
+          username,
+          displayName,
+          birthDate,
+        },
+        {
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+        }
+      );
+      setUser(newUserInfo.data);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <form onSubmit={submitUserInfo}>
+      <Box sx={styles.container}>
+        <Typography variant="h1" sx={styles.title}>
+          Let's get to know a little more about you
+        </Typography>
+        <TextField
```

Comment:
This should be a unique identifier but I was able to post identical usernames onto this field. 

Ideally, there should be some check that queries the db to see if that username already exists, and if so, throw an error and do not redirect yet.

I can do this part later as part of another ticket also, looks like fun lol

Reply:
Yeah I know, I was focused on getting the functionality to work first. I didn't put much validation. I'm open to making a new ticket to tackle that